.github/clusterloader2-release.yml

```diff
@@ -0,0 +1,35 @@
+name: Release ClusterLoader2
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - v*
+  release:
+    types: [published, edited]
```

Comment:
I guess only one of them would do no?

Reply:
removed this since I'm going with tag and path